rollup.config.js

```diff
@@ -147,15 +147,28 @@ if (!buildBundles) {
   };
   const configEsmMin = {
     input: libInput,
-    output: {
-      file: pkg.module.replace('.js', '.min.js'),
-      format: 'es',
-    },
+    output: [
+      {
+        file: pkg.module.replace('.js', '.min.js'),
+        format: 'es',
+      },
+      {
+        file: pkg.main.replace('.js', '.min.js'),
+        format: 'cjs',
+      },
```

Comment:
:warning: I am not sure we need the min version for commonjs as it has to be processed server-side or used by a bundler

Reply:
I can't help here. @aibcmars ?

Reply:
it may make sense to have minified version - if someone wants to pickup just minified file from cdn it will lower over network load impact

I always check what the BIG guys do - react for example serves minified version
https://unpkg.com/browse/react@16.7.0/cjs/

Reply:
good point, thanks